lib/rage/router/dsl.rb

```diff
@@ -98,6 +98,33 @@ def root(to:)
       __on("GET", "/", to, nil, nil)
     end
 
+    #  Register a new route that accepts any HTTP method.
+    # @param path [String] the path for the route handler
+    # @param to [String] the route handler in the format of "controller#action"
+    # @param constraints [Hash] a hash of constraints for the route
+    # @param defaults [Hash] a hash of default parameters for the route
+    # @param via [Symbol, Array<Symbol>] an array of HTTP methods to accept
+    # @example
+    #   match "/photos/:id", to: "photos#show", via: ["get", "post"]
+    # @example
+    #   match "/photos/:id", to: "photos#show", via: :all
+    def match(path, to:, constraints: {}, defaults: nil, via: nil)
```

Comment:
Seems to make more sense to initialize `via` as `all`.

```suggestion
    def match(path, to:, constraints: {}, defaults: nil, via: :all)
```